src/build/utils/map.ts

```diff
@@ -16,3 +16,14 @@ export function addToStringMap(
   const old = map.get(name) || "";
   map.set(name, `${old}\n${value}\n`);
 }
+
+export function addToNestedMap(
+  map: Map<string, Map<string, string>>,
+  name: string,
+  key: string,
+  value: string,
+): void {
+  const nested = map.get(name) || new Map();
```

Comment:
```suggestion
  const nested = map.get(name) ?? new Map();
```
But should not be a big difference.